plugin/trino-blackhole/src/test/java/io/trino/plugin/blackhole/TestBlackHoleSmoke.java

```diff
@@ -346,4 +345,15 @@ private void assertThatQueryReturnsValue(String sql, Object expected, Session se
         assertEquals(value, expected);
         assertEquals(Iterables.getOnlyElement(rows).getFieldCount(), 1);
     }
+
+    private void assertThatQueryDoesntReturnValue(String sql)
+    {
+        assertThatQueryDoesntReturnValue(sql, null);
+    }
+
+    private void assertThatQueryDoesntReturnValue(String sql, Session session)
```

Comment:
in query assertions, session typically goes before sql parameter

also, annotate the sql param with `@Language("SQL")` for syntax highlighting in the IDE